examples/BlackSky-product-example.json

```diff
@@ -0,0 +1,416 @@
+{
+    "products": [
+        {
+            "type": "Product",
+            "stat_version": "stat_version",
+            "stat_extensions": [],
+            "id": "BSKY_STEREO",
+            "title": "BlackSky EO STEREO",
+            "description": "A pair of images collected from different viewing angles for 3D visualization and analysis",
+            "keywords": [
+                "EO",
+                "Stereo"
+            ],
+            "license": "?",
+            "providers": {
+                "name": "BlackSky",
+                "description": "First To Know",
+                "roles": [
+                    "producer"
+                ],
+                "url": "www.blacksky.com"
+            },
+            "parameters": {
+                "eNiirs": {
+                    "description": "NIIRS Score is a function of the GSD, Signal-to-Noise Ratio (SNR), and Relative Edge Response (RER) of an image. This NIIRS constraint will use a predicted SNR and RER based off an aggregation of previously taken and analyzed images.",
+                    "minimum": 3,
+                    "maximum": 5,
+                    "defaultValue": [
+                        3,
+                        5
+                    ]
+                },
+                "target_azimuth": {
+                    "description": "Angle between True North, target, and sub-Sun point (clockwise)",
+                    "minimum": 0,
+                    "maximum": 360.0,
+                    "defaultValue": [
+                        0,
+                        360
+                    ]
+                },
+                "view:sun_elevation": {
+                    "description": "Angle between the target ground plane and Sun. Images collected in low-light conditions have quality outside standard specifications.",
+                    "minimum": 10.0,
+                    "maximum": 90.0,
+                    "defaultValue": [
+                        10,
+                        90
+                    ]
+                },
+                "view:sun_azimuth": {
+                    "description": "Angle between True North, target, and sub-satellite point (clockwise, degrees)",
+                    "minimum": 0,
+                    "maximum": 360.0,
+                    "defaultValue": [
+                        0,
+                        360
+                    ]
+                },
+                "view:off_nadir": {
+                    "description": "Angle between the sub-satellite point, satellite, and target (degrees)",
+                    "minimum": 0.0,
+                    "maximum": 30.0,
+                    "defaultValue": [
+                        0,
+                        30
+                    ]
+                },
+                "view:satellite_elevation": {
```

Comment:
I wonder if there are a couple of values for "type" missing for the "view" extensions, e.g. here but also see other "view:xx" fields